projects/odf_transform/test_odf.py

```diff
@@ -1,4 +1,4 @@
-from write_ctd_ncfile_modules import *
+from write_ctd_ncfile_modules_noOce import *
```

Comment:
should be write_ctd_ncfile_modules_no_oce (mixing of camel case and underscores)

Reply:
Yeah we should rename the module itself, that was just a temporary place holder for demonstration